pkgs/applications/video/kodi-packages/steam-library/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildKodiAddon, fetchFromGitHub, requests, requests-cache, routing }:
+
+buildKodiAddon rec {
+  pname = "steam-library";
+  namespace = "plugin.program.steam.library";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "aanderse";
+    repo = namespace;
+    rev = "v${version}";
+    sha256 = "1d8n8zkprjql0nw5ff752yr994hw2ikd0ny3m9hjr90s4kdykjzr";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    requests-cache
+    routing
+  ];
+
```

Comment:
Is this missing passthru?

Reply:
Not this one. It doesn't provide a library, so no `pythonPath`, and it doesn't need a special `updateScript` because the software just follows simple releases on `github` only.